build.gradle

```diff
@@ -340,3 +340,7 @@ buildScan {
   licenseAgreementUrl = 'https://gradle.com/terms-of-service'
   licenseAgree = 'yes'
 }
+
+wrapper {
+  gradleVersion = '4.7'
```

Comment:
Is this a good idea? Gradle will per default recreate the wrapper with the version that is being executed (so you basically duplicate the setting from `gradle-wrapper.properties`). Overriding that is still possible via command-line `--gradle-version`

Reply:
By specifying the wrapper here, you can just write ./gradlew wrapper, and all the gradle related wrapper files will update with the correct version.  No need to edit them manually.

Reply:
I personally also stopped using `wrapper {}` in `build.gadle` as it is easier to just call the wrapper with `--gradle-version` than edit the value in `build.gradle` and call the wrapper. However, it may be a matter of preferences.

Reply:
I don't see the use case to make recreating the wrapper files convenient. They are under version control deliberately. The only reason to do this is to *change* the version, in which case you won't want the `4.7` you hard coded here.